soc/ti/hercules/tms570lc43x/Kconfig.soc

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 ispace, inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_TI_HERCULES
+	bool
+	select SOC_FAMILY_HERCULES
+	help
+	  TI Hercules family
+
+config SOC_TI_TMS570LC43X
+	bool
+	select SOC_SERIES_TI_HERCULES
+	help
+	  TI TMS570LC43X
+
+config SOC_SERIES
+	default "ti_hercules" if SOC_SERIES_TI_HERCULES
+
+config SOC
+	default "tms570lc43x" if SOC_TI_TMS570LC43X
```

Comment:
as above, needs `ti_` in string or TI_ removing from symbol, must match soc.yml

Reply:
fixed